dd-trace-ot/src/main/java/datadog/opentracing/DDTracer.java

```diff
@@ -670,6 +671,12 @@ public DDSpanBuilder withSpanType(final String spanType) {
       return parent.baggageItems();
     }
 
+    public DDSpanBuilder withLogHandler(final LogHandler logHandler) {
+      assert logHandler != null : "LogHandler must not be null";
+      this.logHandler = logHandler;
```

Comment:
```suggestion
      if(logHandler != null) {
          this.logHandler = logHandler;
      }
```
Now that I think about it, I think this is a better option...  Avoids the risk of harmful exceptions.